dist/t/spec/features/0040_package_spec.rb

```diff
@@ -37,8 +37,7 @@
       click_link('Home Project')
     end
     click_link('Branch Existing Package')
-    sleep 1 # wait for the modal to appear
-    within('#new-package-branch-modal') do
+    within('#new-package-branch-modal', wait: 5) do # wait for the modal to appear
```

Comment:
I think Capybara waits for elements to appear by default, check https://github.com/teamcapybara/capybara?tab=readme-ov-file#asynchronous-javascript-ajax-and-friends. Even if this has nothing to do with AJAX, at end of the section there is an example regarding modals using `find` to wait for some elements to appear.

Reply:
Sure